docs/operate-and-deploy/installation/server-config/security.md

```diff
@@ -817,21 +796,21 @@ cluster to allow ksqlDB to operate:
 
 ```bash
 # Allow ksqlDB to discover the cluster:
-bin/kafka-acls --authorizer-properties zookeeper.connect=localhost:2181 --add --allow-principal User:KSQL1 --allow-host 198.51.100.0 --allow-host 198.51.100.1 --allow-host 198.51.100.2 --operation DescribeConfigs --cluster
+bin/kafka-acls --authorizer-properties bootstrap-server=localhost:2181 --add --allow-principal User:KSQL1 --allow-host 198.51.100.0 --allow-host 198.51.100.1 --allow-host 198.51.100.2 --operation DescribeConfigs --cluster
```

Comment:
`--authorizer-properties` should be removed, and bootstrap port should be 9092 instead of 2181. Same for all other examples which follow.

Reply:
Thanks, @mikebin, how does it look now?